tests/test_cli_compile.py

```diff
@@ -458,8 +458,24 @@ def test_quiet_option(runner):
     with open("requirements", "w"):
         pass
     out = runner.invoke(cli, ["--quiet", "-n", "requirements"])
-    # Pinned requirements result has not been written to output
-    assert "Dry-run, so nothing updated." == out.stderr.strip()
+    # Pinned requirements result has not been written to output.
+    assert b"" == out.stderr_bytes
```

Comment:
Just a little nitpick

```suggestion
    assert not out.stderr_bytes
```